src/pages/overlay/components/buttons/Buttons.tsx

```diff
@@ -27,15 +28,30 @@ export default function Buttons<T extends ButtonsOptions = ButtonsOptions>(props
 
   return (
     <div className={styles.activationButtons}>
-      {optionsWithOnClick.map(option => (
-        <button
+      {optionsWithOnClick.map((option) => (
+        <Tooltip
           key={option.key}
-          onClick={option.onClick}
-          className={classes(option.active && styles.highlighted, option.type === "secondary" && styles.secondary)}
+          text={option.hoverText}
+          ariaLabel={option.title}
+          className={classes(
+            styles.tooltip,
```

Comment:
```suggestion
            styles.btn,
```
No point having two classes doing the same thing